src/gpu/GPU.h

```diff
@@ -0,0 +1,10 @@
+#include "../timestamp.hpp"
+
+#define MAX_MARKERS_PER_READ (1 << 15)
+#define GPU_BATCH_SIZE (1 << 12)
+#define MIN_GPU_BATCH_SIZE (1 << 4)
```

Comment:
Please use a SHASTA prefix fore the above macros, to prevent name collisions.

Reply:
Sure.